media-libs/libwmf/libwmf-0.2.12~git.recipe

```diff
@@ -0,0 +1,143 @@
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libWMFVersion="7.1.4"
+libWMFVersionCompat="$libWMFVersion compat >= ${libWMFVersion%%.*}"
+libWMFLITEVersion="7.0.5"
+libWMFLITEVersionCompat="$libWMFLITEVersion compat >= ${libWMFLITEVersion%%.*}"
+
+PROVIDES="
+	libwmf$secondaryArchSuffix = $portVersion
+	lib:libwmf_0.2$secondaryArchSuffix = $libWMFVersionCompat
+	lib:libwmflite_0.2$secondaryArchSuffix = $libWMFLITEVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libwmf${secondaryArchSuffix}_devel = $portVersion
+	cmd:libwmf_config$commandSuffix
+	cmd:libwmf_fontmap$commandSuffix
+	devel:libwmf$secondaryArchSuffix
+	devel:libwmf_0.2$secondaryArchSuffix = $libWMFVersionCompat
+	devel:libwmflite$secondaryArchSuffix
+	devel:libwmflite_0.2$secondaryArchSuffix = $libWMFLITEVersionCompat
+	"
+REQUIRES_devel="
+	libwmf$secondaryArchSuffix == $portVersion base
+	brotli${secondaryArchSuffix}_devel
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgraphite2$secondaryArchSuffix
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+PROVIDES_tools="
+	libwmf${secondaryArchSuffix}_tools = $portVersion
+	cmd:wmf2eps$commandSuffix
+	cmd:wmf2fig$commandSuffix
+	cmd:wmf2gd$secondaryArcommandSuffixchSuffix
+	cmd:wmf2svg$commandSuffix
+	"
+REQUIRES_tools="
+	libwmf$secondaryArchSuffix == $portVersion base
+	haiku$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libexpat$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgdk_pixbuf_2.0$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage libwmf$secondaryArchSuffix \
+	$libDir/libwmf-0.2.so.$libWMFVersion \
+	$libDir/libwmflite-0.2.so.$libWMFLITEVersion \
+	"$(getPackagePrefix tools)"/bin/wmf2eps \
+	"$(getPackagePrefix tools)"/bin/wmf2fig \
+	"$(getPackagePrefix tools)"/bin/wmf2gd \
+	"$(getPackagePrefix tools)"/bin/wmf2svg
+
+BUILD()
+{
+	autoreconf -fi
+	runConfigure --omit-dirs binDir ./configure \
+		--bindir=$commandBinDir \
+		--with-docdir=$docDir \
+		--with-fontdir=$dataDir/libwmf/fonts \
+		--with-xtrafontmap=$dataDir/libwmf/fonts/fontmap #\
+#		--with-gsfontdir=$fontsDir/otfonts\
+#		--with-gsfontmap=$dataDir/ghostscript/9.55.0/lib/Fontmap.GS
```

Comment:
Still a bit unsure here, documentation isn't installed, so setting this isn't needed maybe, couldn't find any fontmap on my system (ghostscript_gpl and the urw fonts installed) ...

Reply:
From looking at some other recipes at repology I also see that they remove the fonts all together because they should be provided by gsfonts (which afaik doesn't exists in haikuports)?

Reply:
perhaps the URW base fonts (`urw_base35_fonts`) are meant here? They come from Ghostscript IIRC.

Reply:
Thought so too, but those are installed in `$fontsDir/otfonts`, the ones in here are similar to the ones in `$fontsDir/otfonts` and don't provide a "fontmap"?
The extentions of the ones here are `t1` and `afm`, the ones in `$fontsDir/otfonts` are `otf`, then there are `afm` fonts in `$fontsDir/psfonts` ...

Reply:
Otherwise add a TODO.

Reply:
Will add a comment there, thanks!